module1/homework/removeVowels/vowel.cpp

```diff
@@ -0,0 +1,13 @@
+#include "vowel.hpp"
+
+#include <algorithm>
+
+const std::string vowels{"aAeEiIoOuUyY"};
```

Comment:
I think constrexpr here would be better since we know this value before compiling would even start and I would consider moving it to the header since You would like to have variable easy to access.

Reply:
and also create as constexpr char[]